include/glow/Backends/CompiledFunction.h

```diff
@@ -28,6 +30,9 @@ class CompiledFunction {
 
   /// Execute the network.
   virtual void execute() = 0;
+  /// Execute the network and allocate Placeholder memory with given
+  /// Context providing mapping between Placeholder and populated tensor.
```

Comment:
nit: when using Doxygen, you can reference the actual parameter with \p.
E.g., [...] with the given \p ctx. [...]
That also encourages to use fully fleshed names (like context instead of ctx ;)).